docs/source/advanced/plugin-configuration.mdx

```diff
@@ -73,7 +73,7 @@ apollo {
 By default, the Gradle plugin registers a `downloadApolloSchema` task that you can use from the command line:
 
 ```bash
-# --schema is interpreted relative to the current working directory. This example
+# --schema is interpreted relative to the project's root directory (can also be an absolute path). This example
```

Comment:
Something I realized during the holidays (don't ask me why I was thinking about this!) and I'm dumping here before I forget: We have the same issue with `./gradlew convertApolloSchema`. Just opened https://github.com/apollographql/apollo-kotlin/issues/3764